packages/base/src/3dview/mainview.tsx

```diff
@@ -1887,6 +1899,14 @@ export class MainView extends React.Component<IProps, IStates> {
     }
   };
 
+  private _handleExplosionChange = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    const newValue = parseInt(event.target.value, 10);
+    this.setState({ explodedViewFactor: newValue });
+    this._explodedView.factor = newValue;
```

Comment:
Do we need to call again `_setupExplodedView` again after this line?

Reply:
@martinRenou Thanks for your comment. It has been taken into account.